coh/deploy.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: coh
+  labels:
+    app: coh
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: coh
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  template:
+    metadata:
+      name: coh
+      labels:
+        app: coh
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: coh
+          image: wordpress:6.0.2-apache
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 80
+              name: http
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 3
+            httpGet:
+              path: /
+              port: http
+              scheme: HTTP
+            initialDelaySeconds: 30
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 2
+          readinessProbe:
+            failureThreshold: 1
+            httpGet:
+              path: /
+              port: http
+              scheme: HTTP
+            periodSeconds: 30
+            successThreshold: 1
+            timeoutSeconds: 2
+          resources:
+            limits:
+              memory: 256Mi
+            requests:
+              cpu: 100m
+              memory: 128Mi
+          env:
+            - name: WORDPRESS_DB_HOST
+              value: joost.chnet
+            - name: WORDPRESS_DB_USER
+              value: cie_coh
+            - name: WORDPRESS_DB_NAME
+              value: cie_coh
+            - name: WORDPRESS_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: cie_coh-db-password
+            - name: WORDPRESS_DB_PREFIX
+              value: wp_
+            - name: APACHE_LOG_DIR
+              value: /var/log/apache/
+          securityContext:
+            runAsNonRoot: false
+          volumeMounts:
+            - name: coh-storage
```

Comment:
@julian9499 do we need more configuration for the spaceclaim than this?

And how to add the content? SSH into the pod and copy and paste?

Reply:
I think so, but otherwise we can do the same as we did for the study visit which is make it with kubectl